.github/workflows/linux.yml

```diff
@@ -11,6 +11,73 @@ on:
       - stable
 
 jobs:
+  acceptance_test:
+    name: RR <-> PHP <-> Temporal Acceptance tests (Go ${{ matrix.go }}, PHP ${{ matrix.php }}, OS ${{matrix.os}})
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 60
+    strategy:
+      matrix:
+        php: [ "8.4" ]
+        go: [ stable ]
+        os: [ "ubuntu-latest" ]
+    steps:
+      - name: Set up Go ${{ matrix.go }}
+        uses: actions/setup-go@v6 # action page: <https://github.com/actions/setup-go>
+        with:
+          go-version: ${{ matrix.go }}
+
+      - name: Set up PHP ${{ matrix.php }}
+        uses: shivammathur/setup-php@v2 # action page: <https://github.com/shivammathur/setup-php>
+        with:
+          php-version: ${{ matrix.php }}
+          extensions: dom, sockets, grpc, curl, protobuf
+
+      - name: Check out code
+        uses: actions/checkout@v5
+        with:
+          submodules: true
+
+      - name: Install dependencies with composer
+        uses: ramsey/composer-install@v3
+        with:
+          working-directory: tests/acceptance/php-sdk
+
+      - name: Init Go modules Cache # Docs: <https://git.io/JfAKn#go---modules>
+        uses: actions/cache@v4
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: ${{ runner.os }}-go-
+
+      - name: Download binaries
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          cd tests/acceptance/php-sdk
+          vendor/bin/dload get temporal
+          vendor/bin/dload get temporal-tests-server
+
+      - name: Build RoadRunner
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          cd tests/acceptance
+          php-sdk/vendor/bin/dload build -vv
```

Comment:
Replace with `go install` (for the velox) + predefined config in the repo env variables.